src/create_shards.c

```diff
@@ -378,6 +380,8 @@ ParseWorkerNodeFile(char *workerNodeFilename)
 	char workerNodeLine[MAXPGPATH];
 	char *workerFilePath = make_absolute_path(workerNodeFilename);
 	char workerLinePattern[1024];
+	const int WorkerNameIndex = 0;
+	const int WorkerPortIndex = 1;
```

Comment:
Good catch. Yeah, I copy-pasted it. I was going to answer "well, these are `const`, so they are capitalized", but then I realized no where else in `pg_shard` do we capitalize `const` variables like this. Even though it's copied from Citus, this function is essentially a reimplementation (or emulation) of what Citus does.

In short, I'm going to make them lowercase and I agree.